src/In.ProjectEKA.HipService/appsettings.Development.json

```diff
@@ -19,7 +19,14 @@
     "BaseUrl":"http://otp"
   },
   "rabbit": {
-    "HostName": "rabbitmq",
```

Comment:
Is this need to be pushed?

Reply:
Yes.. In development it is supposed to be "localhost"